### DISCUSSION TITLE
174

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#139] [General]
# 潮流周刊第174期 - 新的园区

封面图拍摄来源于周内去逛了下个月要搬的新园区，环境挺好的，甚至还有足球场，非常有大学的感觉，很喜欢，期待后面可以在那儿快乐工作。

https://weekly.tw93.fun/posts/174-%E6%96%B0%E7%9A%84%E5%9B%AD%E5%8C%BA

**Comment by xxholly32** [comment]
杭州的面馆要看地方，一般都也是市井里面的面馆生意好，喜欢爆鳝面的话，市区有家方老大还行（忘记是不是甜口了，不过杭州本帮菜的有些确实会偏甜）

**Reply by tw93** [reply]
哈哈 主要是甜口，其实喜欢的人应该会觉得很不错，上海面也是甜口，吃不太习惯

**Comment by mrdream24** [comment]
是不是原来西溪园区的大学改建的园区？

**Reply by tw93** [reply]
是的是的

**Comment by Dacong-wu** [comment]
请问网飞那个要会员嘛，没用过，这个字体真的很好看

**Reply by tw93** [reply]
需要的，可以去电报群找一些靠谱的拼车，一个季度也就45，甚至比国内会员便宜很多

**Reply by Dacong-wu** [reply]
那真不错

**Comment by image-gif** [comment]
之前去西溪A,B区已经感觉非常像大学了，没想到这下直接安排上了，哈哈

**Comment by QuakeWang** [comment]
以后的周刊期待可以多放一点好吃的 哈哈哈。芒种的拌川也挺不错的~~

**Reply by tw93** [reply]
可以可以，会的会的
